### PR TITLE
oc get templates: print only the first line of description

### LINKED ISSUE #11765: Get output for templates with newlines in their descriptions is broken
```
./oc get templates -n openshift
NAME                                     DESCRIPTION                                                PARAMETERS     OBJECTS
cakephp-mysql-example                    An example CakePHP application with a MySQL database       19 (4 blank)   7
dancer-mysql-example                     An example Dancer application with a MySQL database        18 (5 blank)   7
django-psql-example                      An example Django application with a PostgreSQL database   17 (5 blank)   7
jenkins-ephemeral                        Jenkins service, without persistent storage.
WARNING: Any data stored will be...      6 (1 generated)   6
jenkins-persistent                       Jenkins service, with persistent storage.
You must have persistent volumes av...   7 (1 generated)                                                                    7
jenkins-pipeline-example                 This example showcases the new Jenkins Pipeline integration in OpenShift, whi...   5 (4 generated)   8
mariadb-ephemeral                        MariaDB database service, without persistent storage. WARNING: Any data store...   6 (2 generated)   2
mongodb-ephemeral                        MongoDB database service, without persistent storage. WARNING: Any data store...   7 (3 generated)   2
mysql-ephemeral                          MySQL database service, without persistent storage. WARNING: Any data stored...    6 (2 generated)   2
nodejs-mongodb-example                   An example Node.js application with a MongoDB database                             16 (4 blank)      7
postgresql-ephemeral                     PostgreSQL database service, without persistent storage. WARNING: Any data st...   6 (2 generated)   2
rails-postgresql-example                 An example Rails application with a PostgreSQL database                            20 (4 blank)      7
```

The "WARNING" lines are broken.  The printer for templates should trim on newline.

### PR DESCRIPTION
**Pull request by mmilata**
Fixes #11765. 
@bparees @smarterclayton PTAL

### CODE REVIEW

File: pkg/cmd/cli/describe/printer.go [hunk 1]
```diff
@@ -178,6 +178,10 @@ func printTemplate(t *templateapi.Template, w io.Writer, opts kctl.PrintOptions)
 	if t.Annotations != nil {
 		description = t.Annotations["description"]
 	}
+	// Only print the first line of description
+	if strings.ContainsRune(description, '\n') {
```
Comment (mfojtik) [inline pkg/cmd/cli/describe/printer.go]: if lines := strings.Split(description, "\n"); len(lines) > 1 {
  description = lines[0] + "..."
}

Comment (liggitt) [inline pkg/cmd/cli/describe/printer.go]: Use SplitN(…, …, 2)

**Comment by mmilata** [comment]
Updated.

**Comment by bparees** [comment]
lgtm [merge]

**Comment by bparees** [comment]
can't access the failing job but i'm sure it's a flake.
[merge]

**Comment by bparees** [comment]
[test]

**Pull request merged**